server/controllers/forgotPassword/verifyUrl.js

```diff
@@ -0,0 +1,32 @@
+import forgotPwUser from '../../models/forgotPwUserModels.js';
+import moment from 'moment';
+import jwt from 'jsonwebtoken';
+
+export const verifyUrl = async (req, res) => {
+  const timeAccessedToUrlAt = moment().utc().valueOf();
+  const hasheduserid = req.params.hasheduserid;
+  console.log(`hashed user id in verify url: ${hasheduserid}`);
+  const decodedUserId = jwt.verify(hasheduserid, process.env.TOKEN_SECRET);
+  try {
+    const forgotPasswordUser = await forgotPwUser.findOne({
+      userId: decodedUserId,
+    });
+    console.log(forgotPasswordUser);
+    // 900000ms = 15min see how much time passed
+    const timePassed = forgotPasswordUser
+      ? timeAccessedToUrlAt - forgotPasswordUser.urlCreatedAt
+      : 900000;
```

Comment:
Yeah, because there is the conditional 
` if (forgotPasswordUser && timePassed < 900000) { `
      `res.status(201).json({ message: 'url still valid!' });`
      `console.log('url still valid!');`
    `} else {`
right after the variable
it should be okay without the ternary operator. I'll double check later just in case but I'm going to remove it if it's not needed. Thank you!